proposals/description/fip_60.ts

```diff
@@ -0,0 +1,153 @@
+      values: '0',
+      method: '_deployMarket(bool,bytes,uint256)',
+      arguments: [
+        false,
+        '0x00000000000000000000000006cb22615ba53e60d67bf6c341a0fd5e718e1655000000000000000000000000c54172e34046c1653d1920d40333dd358c7a1af4000000000000000000000000bab47e4b692195bf064923178a90ef999a15f8190000000000000000000000000000000000000000000000000000000000000120000000000000000000000000000000000000000000000000000000000000016000000000000000000000000067db14e73c2dce786b5bbbfa4d010deab4bbfcf900000000000000000000000000000000000000000000000000000000000001a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001f46656952617269204645492d33437276204d6574616d706f6f6c204675736500000000000000000000000000000000000000000000000000000000000000000b664645492d334372762d380000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+        '600000000000000000'
+      ],
+      description: 'Add FEI-3Crv to FeiRari'
+    },
+    {
+      target: 'rariPool8Comptroller',
+      values: '0',
+      method: '_deployMarket(bool,bytes,uint256)',
+      arguments: [
+        false,
+        '0x0000000000000000000000003d1556e84783672f2a3bd187a592520291442539000000000000000000000000c54172e34046c1653d1920d40333dd358c7a1af4000000000000000000000000bab47e4b692195bf064923178a90ef999a15f8190000000000000000000000000000000000000000000000000000000000000120000000000000000000000000000000000000000000000000000000000000016000000000000000000000000067db14e73c2dce786b5bbbfa4d010deab4bbfcf900000000000000000000000000000000000000000000000000000000000001a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001e466569526172692047656c61746f204645492d444149204c5020467573650000000000000000000000000000000000000000000000000000000000000000001066472d554e492d4645492d4441492d38000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
+        '600000000000000000'
+      ],
+      description: 'Add FEI-DAI G-UNI to FeiRari'
+    },
+    {
+      target: 'tribalChief',
+      values: '0',
+      method: 'add(uint120,address,address,(uint128,uint128)[])',
+      arguments: ['250', '{d3StakingTokenWrapper}', '0x0000000000000000000000000000000000000000', [[0, 10000]]],
+      description: 'Add d3 to TribalChief'
+    },
+    {
+      target: 'tribalChief',
+      values: '0',
+      method: 'add(uint120,address,address,(uint128,uint128)[])',
+      arguments: ['250', '{fei3CrvStakingtokenWrapper}', '0x0000000000000000000000000000000000000000', [[0, 10000]]],
+      description: 'Add FEI-3Crv to TribalChief'
+    },
+    {
+      target: 'tribalChief',
+      values: '0',
+      method: 'add(uint120,address,address,(uint128,uint128)[])',
```

Comment:
for really complex dao actions like this, would appreciate it if you added comments showing what each argument is supposed to represent

Reply:
adding